web/src/App.less

```diff
@@ -134,6 +126,11 @@
   box-sizing: border-box;
   margin: 0 auto;
   position: relative;
+  box-shadow: 0 0 20px rgb(0 0 0 / 20%);
```

Comment:
Please don't modify class `.login-panel`. It't will cause many other problem that incompatible with other feature about the custom login page ui. For beatifying the default login page.  As far as possible just modify the class `.login-form`

![image](https://user-images.githubusercontent.com/71440988/200284415-f5f65efa-4e5a-4914-88e3-393fdacb5a43.png)

Reply:
Looking into it.

Reply:
Hi @leo220yuyaodog I've double checked but it seems I didn't modify `.login-panel`? How to reproduce your result shown in the screenshot?

Reply:
> Hi @leo220yuyaodog I've double checked but it seems I didn't modify `.login-panel`. How to reproduce your result shown in the screenshot? What I have changed is `.login-content` only.

Custom the application UI follow the document https://casdoor.org/docs/application/ui-customization

Reply:
> Custom the application UI follow the document https://casdoor.org/docs/application/ui-customization

I see the problem here. Does the style filled in Form CSS in casdoor overwrite the default style in `.login-panel`? If that is the case, can we just assign `.login-panel` a more unified default style?

Reply:
@imchell , @leo220yuyaodog wrote this part of code and is more familar with it. I think it's more suitable for you to create an issue instead and let him fix it.